README.md

```diff
@@ -5,3 +5,40 @@ Notepad++ supports around 80 programming languages for syntax highlighting & fol
+   * The author's name.  Example: `STL_udl.byPryrt.xml`, if Pryrt was not creative enough to come up with a better description of his UDL for the STL syntax highlighting.
+   * Use underscores or hyphens or periods to separate words, not spaces.
+3. **recommendation**: each submitted UDL file should only contain one lanugage it is defining.  This will make it easier for users to only download what they need.
+   * A possible exception might be if you are bundling multiple UDL for related languages all using the same theme.  Example: `3dModeling_bundle_forChocoTheme_includes_STL_OBJ_3DS.udl.xml` would implement highlighting rules for the three 3D modeling formats of STL, OBJ, and 3DS, all for the same Choco theme.
+4. **recommendation**: if your UDL file only contains one language, the `display-name` attribute in the JSON file (described below) should have the same value as the `<UserLang name="...">` inside your definition file.  This will keep the name in the **Language** menu the same as the name that was shown in the download tool (coming soon).
+5. **recommendation**: in your Pull Request, please provide a link to a public description of the language your UDL is based on (which will help to establish the general-interest nature of the UDL), as well as a link to an example file in that language (so that the UDL can be verified as functional).
+   * If you have an example file, you can upload it to the `UDL samples` folder of the repository. Please have this file use the same name as your UDL definition file, but with the appropriate file extension, rather than `.xml`.  Example: `UDLs\STL_udl.byPryrt.xml` would have a corresponding example file `UDL samples\STL_udl.byPryrt.stl`.
+
+When you make a submission, \[to be decided: you should | we will\] edit the `udl-list.json` file.  
+* The `id-name` attribute will match the name of the UDL file, without the `.xml`.
+* The `display-name` attribute will match the `<UserLang name="...">` from the submitted UDL file, if possible.
+* The `version` attribute can have a `v1.00`-style version or just a last-edited date like `2020-Jan-12`.
+* The `repository` attribute will be left blank for UDL files uploaded to this repository; if the UDL resides in an external repository (or if you want to add a link to the original source file, but still keep a copy in this repository), use this attribute to hold the full URL of the external file
+* The `description` attribute is an optional description of the UDL language (it could be the based on the `id-name` or `display-name`, but with spaces instead of underscores, hyphens, and/or periods).
+* The `author` attribute should be your name or GitHub username.
+* The `homepage` attribute could be a link to your GitHub repository for the UDL language, or your general GitHub or other personal homepage.
```

Comment:
"personal homepage" means it could be everything.
I would restrict it to be the link about the UDL(in GitHub or somewhere-else), or contributors' GitHub page.